qemu/tests/arm_cpu_test_clusters.py

```diff
@@ -0,0 +1,43 @@
+import random
+
+from virttest import error_context
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check guest gets correct multiple vcpu clusters
+
+    1) Boot guest with options: -smp n,clusters=2x...
+    2) Check cpu clusters(only for Linux guest)
+
+    :param test: QEMU test object.
+    :param params: Dictionary with test parameters.
+    :param env: Dictionary with the test environment.
+    """
+    vm_name = params['main_vm']
+    vcpu_clusters_list = [2, 4]
+    params['vcpu_clusters'] = random.choice(vcpu_clusters_list)
+    params['start_vm'] = 'yes'
+    env_process.preprocess_vm(test, params, env, vm_name)
```

Comment:
Seems no contains some prepare steps, I think you can use `vm.create(params=params)` directly.

Reply:
corrected